plugins/ledmatrix-music/ytm_client.py

```
@@ -80,25 +100,25 @@ def on_state_update(data):
                 title = data.get('video', {}).get('title', 'N/A') if isinstance(data, dict) else 'N/A'
                 author = data.get('video', {}).get('author', 'N/A') if isinstance(data, dict) else 'N/A'
                 is_playing = (data.get('player', {}).get('trackState') == 1) if isinstance(data, dict) else None
-                logging.debug(f"[YTMClient] socket event received: title='{title}', artist='{author}', is_playing={is_playing}")
+                self._log.debug(f"[YTMClient] socket event received: title='{title}', artist='{author}', is_playing={is_playing}")
             except Exception:
                 pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)